conf/base.config

```
@@ -1,36 +1,14 @@
 /*
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Ensembl/datascout Nextflow base config file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    A 'blank slate' config file, appropriate for general use on most high performance
-    compute environments. Assumes that all software is installed and available on
-    the PATH. Runs in `local` mode - all jobs will be run on the logged in environment.
-----------------------------------------------------------------------------------------
+Nextflow base configuration file
 */
 
 process {
     
-    resourceLimits = [
-        cpus: 1,
-        memory: 25.GB,
-        time: '6.h'
-    ]
-    
-    
-    // TODO nf-core: Check the defaults for all processes
-    cpus   = {  1    * task.attempt }
-    memory = {  25.GB * task.attempt }
-    time   = {  6.h  * task.attempt }
-  
-    
     errorStrategy = { task.exitStatus in ((130..145) + 104) ? 'retry' : 'finish' }
```

```suggestion
    cpus   = { 1      * task.attempt }
    memory = { 6.GB   * task.attempt }
    time   = { 4.h    * task.attempt }

    errorStrategy = { task.exitStatus in ((130..145) + 104 + 175) ? 'retry' : 'finish' }
```

i would put this back, nf-core style

---

So when I was trying to run things these requirements seemed to be overwriting everything else even when I used a profile. Like blah blah process needs 4 cpus and only 1 provided. Did I do something wrong?

---

Usually it's a configuration clash/problem, when trying to combine different configs and things overwrite each other..i.e. test,slurm,local profiles etc.